src/OpenTK.Graphics/OpenGL/GL.Manual.cs

```diff
@@ -20,5 +20,19 @@ public static void ShaderSource(uint shader, string str)
             GL.ShaderSource(shader, 1, (byte**)&str_iptr, length);
             Marshal.FreeCoTaskMem(str_iptr);
         }
+
+        public static void GetShaderInfoLog(uint shader, out string info)
```

Comment:
Can this be moved into some base class of some sort, otherwise this piece of code exists 3 times

in
`OpenGL/Compatibility/GL.Manual.cs`
`OpenGL/GL.Manual.cs`
`OpenGLES3/GL.Manual.cs`

Reply:
It may be able to be moved at some point. However atm this is not possible. But this is also the way it worked before for manual overloads and with this new system, less manual overloads should be necessary.

Btw, the reason it cant simply be moved into one base class is because
1: the GL class is static, and no inheritance works yet on static classes.
2: All the methods are not necessarily part of every version. There is a fourth version of these files too. But im assuming there is no shader info logs in gles1 as that one didnt recieve this overload.

Reply:
What about `GL.Manual.Base.cs`. `...of some sort` could also mean a static helper method maybe, not necessarily a base class?